src/store/actions.js

```diff
@@ -502,30 +502,19 @@ export default {
 			newUid: uid,
 		})
 	},
-	deleteMessage({getters, commit}, envelope) {
-		const folder = getters.getFolder(envelope.accountId, envelope.folderId)
-		commit('removeEnvelope', {
-			accountId: envelope.accountId,
-			folderId: envelope.folderId,
-			id: envelope.id,
-		})
+	deleteMessage({getters, commit}, accountId, folderId, id) {
```

Comment:
For consistency with the other actions I'd suggest to also use the object destruction like `{accountId, folderId, id}` for `deleteMessage`

Reply:
Cool! 👍 In that case, do you have any preference on how to pass in the arguments?

#### Explicitly picking the relevant properties
```
this.$store.dispatch('deleteMessage', {
	accountId: this.message.accountId,
	folderId: this.message.folderId,
	id: this.message.id,
})
```
#### Sending in the whole object
```
this.$store.dispatch('deleteMessage', this.message)
```
#### Or using something like Ramda's `pick` to make a copy
```
this.$store.dispatch('deleteMessage', pick(['accountId', 'folderId', 'id'], this.message))
```

I think there's some maintainability and grokability value in the explicitness of the first and last options, but the middle option looks cleaner 😅

Reply:
I like all of them :see_no_evil: 

The first one is more consistent to how we pass the objects to other actions right now, though.

Reply:
Went with the first one before I saw this message, so guess that's the one I'm going with for now 😅